.github/workflows/rspec.yml

```diff
@@ -0,0 +1,31 @@
+name: rspec tests
+
+on: [pull_request]
+
+jobs:
+  rspec:
+    name: Rspec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_DB: pangolin_test
+          POSTGRES_USER: pangolin
+          POSTGRES_PASSWORD: postgres
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-project
+
+      - uses: ./.github/actions/setup-database
+
+      - name: Run rspec
+        run: bundle exec rspec
```

Comment:
[suggestion] passing the `DATABASE_URL` environment variable will let the tests see the db